frame/support/procedural/src/construct_runtime/expand/metadata.rs

```diff
@@ -95,11 +95,31 @@ pub fn expand_runtime_metadata(
 				// `Deref` needs a reference for resolving the function call.
 				let rt = #runtime;
 
+				let ty = #scrate::scale_info::meta_type::<#extrinsic>();
+				let address_ty = #scrate::scale_info::meta_type::<
+						<<#extrinsic as #scrate::sp_runtime::traits::Extrinsic>::SignaturePayload as #scrate::sp_runtime::traits::SignaturePayload>::SignatureAddress
+					>();
+				let call_ty = #scrate::scale_info::meta_type::<
+					<#extrinsic as #scrate::sp_runtime::traits::Extrinsic>::Call
+					>();
+				let signature_ty = #scrate::scale_info::meta_type::<
+						<<#extrinsic as #scrate::sp_runtime::traits::Extrinsic>::SignaturePayload as #scrate::sp_runtime::traits::SignaturePayload>::Signature
+					>();
+				let extra_ty = #scrate::scale_info::meta_type::<
+						<<#extrinsic as #scrate::sp_runtime::traits::Extrinsic>::SignaturePayload as #scrate::sp_runtime::traits::SignaturePayload>::SignatureExtra
+					>();
+				let len_ty = #scrate::scale_info::meta_type::<#scrate::codec::Compact<u32>>();
```

Comment:
Based on the comment of @jsdw we should not add this.

Reply:
I've removed the `len_ty` form the V15 conversion, thanks!